source/documentation/runbooks/008-cadet-integration.html.md.erb

```diff
@@ -0,0 +1,68 @@
+---
+owner_slack: "#data-catalogue"
+title: Configure DBT (Create a Derived Table) to send metadata to the catalogue
+last_reviewed_on: 2024-06-04
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+The data catalogue uses DBT as a source of metadata about the Analytical Platform.
+
+By default, all models and sources will be ingested into the Datahub catalogue, but they will not be shown in the Find MOJ Data service.
```

Comment:
We'll need to amend the ingestion recipe to make this the case

Reply:
Got a PR for that here https://github.com/moj-analytical-services/create-a-derived-table/pull/1568

Reply:
Our ingestion recipe also needs an amendment so we pull in all tables I mean.

Reply:
Ah right - I'll follow up on that

Reply:
Follow up ticket: https://github.com/ministryofjustice/data-catalogue/issues/145

Reply:
Just need to remove the allow regex block

Reply:
@LavMatt I've included that in the above ticket ☝🏻